light-clients/berachain-light-client/Cargo.toml

```diff
@@ -32,11 +32,13 @@ tendermint-light-client = { workspace = true, features = ["library"] }
 tendermint-verifier     = { workspace = true }
 thiserror               = { workspace = true }
 
-ethereum-verifier  = { workspace = true }
-hex-literal        = { workspace = true }
-ics008-wasm-client = { workspace = true }
-ics23              = { workspace = true }
-unionlabs          = { workspace = true }
+berachain-light-client-types.workspace  = true
+ethereum-verifier                       = { workspace = true }
+hex-literal                             = { workspace = true }
+ics008-wasm-client                      = { workspace = true }
+ics23                                   = { workspace = true }
+tendermint-light-client-types.workspace = true
```

Comment:
let's keep `{ workspace = true }` convention

Reply:
im going to push a formatter change once all of this is done to fix this automatically for us, it's too much work to automatically maintain